flink-table/flink-table-common/src/main/java/org/apache/flink/table/annotation/StateHint.java

```diff
@@ -70,4 +70,25 @@
      * or provide hints for the reflection-based extraction of the data type.
      */
     DataTypeHint type() default @DataTypeHint();
+
+    /**
+     * The time-to-live (TTL) duration that automatically cleans up the state entry.
+     *
+     * <p>It specifies a minimum time interval for how long idle state (i.e., state which was not
+     * updated by a create or write operation) will be retained. State will never be cleared until
+     * it was idle for less than the minimum time, and will be cleared at some time after it was
+     * idle.
+     *
+     * <p>Use this for being able to efficiently manage an ever-growing state size or for complying
+     * with data protection requirements.
+     *
+     * <p>The cleanup is based on processing time, which effectively corresponds to the wall clock
+     * time as defined by {@link System#currentTimeMillis()}).
+     *
+     * <p>The provided string must use Flink's duration syntax (e.g., "3 days", "45 min", "3 hours",
+     * "60 s"). If no unit is specified, the value is interpreted as milliseconds. The TTL setting
+     * on a state entry has higher precedence than the global state TTL configuration for the entire
+     * pipeline.
```

Comment:
would it make sense to put here a link to parser method `org.apache.flink.util.TimeUtils#parseDuration` which has more examples in it`s javadoc?